client/src/components/userprofiles/ReactivateUser.jsx

```diff
@@ -0,0 +1,47 @@
+import { useEffect, useState } from "react";
+import { changeIsActiveStatus, getProfiles } from "../../managers/userProfileManager.js";
+import { Button, Card, CardBody, CardTitle } from "reactstrap";
+import { Link } from "react-router-dom";
+
+export default function ReactivateUser()
+{  const [userprofiles, setUserProfiles] = useState([]);
```

Comment:
Move state onto a new line, rather than being on the same line as the opening curly bracket